ddtrace/appsec/_handlers.py

```diff
@@ -161,19 +163,31 @@ def _on_lambda_start_response(
 
 async def _on_asgi_request_parse_body(receive, headers):
     if asm_config._asm_enabled:
+        more_body = True
+        body_parts = []
         try:
-            data_received = await receive()
+            while more_body:
+                print("Waiting for ASGI body data", file=sys.stderr)
+                data_received = await asyncio.wait_for(receive(), asm_config._fast_api_async_body_timeout)
+                print(f"Receiving for ASGI body data {data_received}", file=sys.stderr)
```

Comment:
Maybe these should be logging calls instead of bare print statements

Reply:
Oh this should be removed, this was for debug only during dev

Reply:
Removed, sorry!